game-core/src/main/java/games/strategy/engine/chat/IChatController.java

```diff
@@ -30,6 +30,9 @@
    */
   void leaveChat();
 
+  /**
+   * A tag associated with a chat participant indicating the participant's role.
```

Comment:
A rename would seem more appropriate of `Tag` -> `ParticipantRole`

That does not seem accurate though 'none' is not the role of a non-moderator. So I think the real fix is to rename this to `ModeratorTag`. Then I would gear the comment towards when and when we use the enum value.

Reply:
The `Tag` enum is used in the signature of RMI methods.  I don't think it can be renamed without breaking network compatibility.

Reply:
Add a TODO that we can/should rename it better? This way a next person won't have to think about what a better name is, and would know why we did not change it. 

On the plus side, network compatibility is the easiest for us to fix, particularly now that we do not have the engine upgrade bug in the wild.

Reply:
4bce5ac9b